client/src/__tests__/EntryForm.test.jsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import EntryForm from '../components/EntryForm';
+
+test('HomePage renders child components', () => {
+  render(
+    <Router>
+        <EntryForm />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!